helm_deploy/templates/deployment.yaml

```diff
@@ -156,7 +156,16 @@ spec:
                 secretKeyRef:
                   name: azure-secret
                   key: app_client_secret
-
+            - name: SIDEKIQ_USERNAME
```

Comment:
An ignorable naming nit, but this is specific to the webui aspect of sidekiq only. 
SIDEKIQ_WEB_UI_USERNAME|SIDEKIQ_WEBUI_USERNAME. 
SIDEKIQ_WEB_UI_PASSWORD|SIDEKIQ_WEBUI_PASSWORD.